isis/cmake/updateBuildPlugins.sh.in

```diff
@@ -0,0 +1,7 @@
+> ${CMAKE_BINARY_DIR}/lib/Camera.plugin
```

Comment:
I pulled down you branch, and it is working. Is it possible to generalize the fix so it works for the other plugins?
> ls -1 lib/*lugin
lib/AtmosModel.plugin
lib/AutoReg.plugin
lib/Camera.plugin
lib/InterestOperator.plugin
lib/NormModel.plugin
lib/PhotoModel.plugin
lib/PolygonSeeder.plugin
lib/Projection.plugin

Reply:
@scsides I'm not sure, I don't see why it couldn't work for other plugins.